.changeset/polite-snails-sip.md

```diff
@@ -0,0 +1,13 @@
+---
+'@astrojs/starlight-docsearch': minor
+'@astrojs/starlight-tailwind': minor
```

Comment:
I messed up at one point, so we’re accidentally already in major version territory for the Tailwind package 😄 (it’s currently at v2.0.3)

```suggestion
'@astrojs/starlight-tailwind': major
```

Reply:
Good catch, I totally missed that :thumbsup: